backend/src/main/java/tqs/electro/electro/controllers/StationController.java

```diff
@@ -13,8 +12,11 @@
 @RequestMapping("/station")
```

Comment:
```suggestion
@RequestMapping("/backend/station")
```

do this so it works with the proxy